src/orderly/read.py

```diff
@@ -0,0 +1,65 @@
+import ast
+
+
+def orderly_read(path):
+    with open(path) as f:
+        src = ast.parse(f.read())
```

Comment:
```suggestion
    src = ast.parse(path.read_text())
```

Assuming `path` is always a `pathlib.Path` and not a string, which as far as I can tell is true